src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.moyo__logo {
+  margin-top: 5px;
```

Comment:
Be consistent with your *vertical* margins. It's a good practice to use either top or bottom margin, but not both, to avoid collapsing margins issues. In this case, you have set a `margin-top` for `.moyo__logo`, make sure this is consistent with the rest of your vertical margins in the design.